addon/components/bourbon-text-field.js

```diff
@@ -22,8 +22,8 @@ export default TextField.extend({
   isFocused: false,
 
   focusedElementObserver: observer('autofocus', function() {
-    if (this.get("autofocus")) {
-      this.set('isFocused', true)
+    if (this.get('autofocus')) {
```

Comment:
Sorry i meant that you can change this whole function to be one line only.

Reply:
```
focusedElementObserver: observer('autofocus', function() { this.set('isFocused', this.get('autofocus')) })
```

Reply:
or like. the function should be on a separate line for readability but yeah
the actual part can be reduced

Reply:
oh..ok.  :) will update